.github/release-drafter-config.yml

```diff
@@ -0,0 +1,16 @@
+name-template: '$RESOLVED_VERSION'
+
+tag-template: '$RESOLVED_VERSION'
+
+categories:
+  - title: 'Features ⭐'
+    label: 'type : feature'
+  - title: 'Chores 🧹'
+    label: 'type : chore'
+  - title: 'Bugs 🐞'
```

Comment:
Optional: Put the emojis in front

Hahahh, sorry. Just think it would look better 😂